packages/cubejs-cockroach-driver/test/CockroachDBRunner.ts

```diff
@@ -0,0 +1,174 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import pgPromise from 'pg-promise';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { DockerComposeEnvironment, Wait } from 'testcontainers';
+import path from 'path';
+import { BaseDbRunner } from './BaseDBRunner';
+
+process.env.TZ = 'GMT';
+
+export class CockroachDBRunner extends BaseDbRunner {
```

Comment:
Could you please explain the sense of split `dbRunner` on two classes in your case? Would it be better to stay with the single class?